publications-2022.bib

```diff
@@ -83,4 +83,68 @@ @article{orlando:2022
          journal = {International Journal for Numerical Methods in Fluids}
 }
 
+@InProceedings{ThiWi2022ICOSAHOM,
+  author={J. P. Thiele, T. Wick},
+  title={Space-time error control using a partition-of-unity dual-weighted residual method applied to low mach number combustion.},
+  booktitle={Proceedings of ICOSAHOM, 2020+1},
+  year={2022},
+  pages={accepted},
+}
+
+@misc{https://doi.org/10.48550/arxiv.2203.14643,
+  author = {Khimin, Denis and Steinbach, Marc C. and Wick, Thomas},
+  title = {Computational performance studies for space-time phase-field fracture optimal control problems},
+  doi = {10.48550/ARXIV.2203.14643},
+  url = {https://arxiv.org/abs/2203.14643},
+  keywords = {Optimization and Control (math.OC), Numerical Analysis (math.NA), FOS: Mathematics, FOS: Mathematics, 74R10, 65N30, 49M15, 49K20, 35Q74},
+  publisher = {arXiv},
+  year = {2022},
+  copyright = {Creative Commons Attribution Non Commercial No Derivatives 4.0 International}
+}
+
+@article{diehl2022comparative,
```

Comment:
Can you update the entry according to https://link.springer.com/article/10.1007/s00466-022-02147-0